_posts/2022/2022-07-25-lessons-from-the-pivot.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Lessons From a Startup Pivot"
+description: "Depending on SAAS, PAAS, and IAAS is great, but it can hurt when you should have turned left at Albuquerque if you haven't insulated yourself from some of those choices."
+tags: [aspnetcore, azure, cloud]
+excerpt_image: https://user-images.githubusercontent.com/19977/180076597-6fbdb672-6539-43fd-b10d-1a1e667c8d8b.png
+---
+
+Building a startup is easy. You file some paperwork and bam! You're a startup!
+
+Building a start-up that's sustainable and can pay everyone a nice salary, on the other hand, is very tough. Today, as my company, [A Serious Business, Inc.](https://www.aseriousbusiness.com/) launches [Abbot on Product Hunt](https://www.producthunt.com/posts/abbot), I thought I'd reflect on some of the lessons learned in the past couple years.
+
+[Abbot](https://ab.bot/) is a Slack App (and bot) that helps teams keep track of customer conversations in Slack.
+
+Some of you might be thinking, "Wait! I thought Abbot was [ChatOps as a service](https://news.ycombinator.com/item?id=27974077)?" Indeed, that was our original goal, but we found it tough to to sell. Conversations often would go like this:
+
+> them: What problem do you solve?
+> us: What problems do you have? Abbot can help with them all!
+> them: *snoring*
+
+Yeah, not too effective as a sales pitch.
+
+## Lesson: Articulate a specific problem your product solves
```

Comment:
I feel like "build something people want" is well trodden advice. This section could probably be cut if you're looking to trim length.